src/main/scala/beam/physsim/jdeqsim/JDEQSimRunner.scala

```diff
@@ -78,14 +78,15 @@ class JDEQSimRunner(
 
     val maybeEventWriter = if (writeEvents) {
       val writer = PhysSimEventWriter(beamServices, jdeqsimEvents)
+      beamServices.matsimServices.addControlerListener(writer)
```

Comment:
@dimaopen could you, please, just add comment the reason behind adding the writer, so it won't be removed accidentally?